docker-for-mac/install.md

```diff
@@ -22,11 +22,12 @@ If you have not already done so, please install Docker for Mac. You can download
 installers from the Stable or beta channel.
 
 Both Stable and Edge installers come with <a
-href="https://github.com/moby/moby/blob/master/experimental/README.md">
+href="https://github.com/docker/docker-ce/blob/master/components/cli/experimental/README.md">
 experimental features in Docker Engine</a> enabled by default and configurable
-on [Docker Daemon preferences](/docker-for-mac/index.md#daemon-experimental-mode) for
-experimental mode. We recommend that you disable experimental features for
-apps in production.
+on [Docker Daemon
+preferences](/docker-for-mac/index.md#daemon-experimental-mode) for experimental
+mode. We recommend that you disable experimental features for apps in
+production.
```

Comment:
But you wouldn't use D4M for production, so I am unclear on why you would need to state this here.

Reply:
well, we say it everywhere as a request from dev .. I'll leave it for now and if you guys want to change it later, you can